Fix testcontainers

### Problem Description
`StartupShutdownServiceTest#testHistoryTablesMatch` fails in the CI
```
java.lang.IllegalStateException: Ryuk failed to start

	at org.testcontainers.utility.ResourceReaper.start(ResourceReaper.java:156)
	at org.testcontainers.DockerClientFactory.client(DockerClientFactory.java:219)
	at org.testcontainers.DockerClientFactory$1.getDockerClient(DockerClientFactory.java:101)
	at com.github.dockerjava.api.DockerClientDelegate.authConfig(DockerClientDelegate.java:107)
	at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:316)
	at de.symeda.sormas.backend.common.StartupShutdownServiceTest.testHistoryTablesMatch(StartupShutdownServiceTest.java:121)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
Caused by: com.github.dockerjava.api.exception.InternalServerErrorException: Status 500: i/o timeout

	at org.testcontainers.shaded.com.github.dockerjava.core.DefaultInvocationBuilder.execute(DefaultInvocationBuilder.java:247)
	at org.testcontainers.shaded.com.github.dockerjava.core.DefaultInvocationBuilder.get(DefaultInvocationBuilder.java:202)
	at org.testcontainers.shaded.com.github.dockerjava.core.DefaultInvocationBuilder.get(DefaultInvocationBuilder.java:74)
	at org.testcontainers.shaded.com.github.dockerjava.core.exec.InspectContainerCmdExec.execute(InspectContainerCmdExec.java:31)
	at org.testcontainers.shaded.com.github.dockerjava.core.exec.InspectContainerCmdExec.execute(InspectContainerCmdExec.java:13)
	at org.testcontainers.shaded.com.github.dockerjava.core.exec.AbstrSyncDockerCmdExec.exec(AbstrSyncDockerCmdExec.java:21)
	at org.testcontainers.shaded.com.github.dockerjava.core.command.AbstrDockerCmd.exec(AbstrDockerCmd.java:35)
	at org.testcontainers.shaded.com.github.dockerjava.core.command.InspectContainerCmdImpl.exec(InspectContainerCmdImpl.java:52)
	at org.testcontainers.utility.ResourceReaper.lambda$start$0(ResourceReaper.java:143)
	at org.testcontainers.shaded.org.awaitility.core.AbstractHamcrestCondition.lambda$new$0(AbstractHamcrestCondition.java:48)
	at org.testcontainers.shaded.org.awaitility.core.ConditionAwaiter$ConditionPoller.call(ConditionAwaiter.java:232)
	at org.testcontainers.shaded.org.awaitility.core.ConditionAwaiter$ConditionPoller.call(ConditionAwaiter.java:219)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at org.testcontainers.shaded.org.awaitility.core.SameThreadExecutorService.execute(SameThreadExecutorService.java:79)
	at java.base/java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:140)
	at org.testcontainers.shaded.org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:98)
	at org.testcontainers.shaded.org.awaitility.core.AbstractHamcrestCondition.await(AbstractHamcrestCondition.java:86)
	at org.testcontainers.shaded.org.awaitility.core.ConditionFactory.until(ConditionFactory.java:939)
	at org.testcontainers.shaded.org.awaitility.core.ConditionFactory.until(ConditionFactory.java:645)
	at org.testcontainers.shaded.org.awaitility.core.ConditionFactory.until(ConditionFactory.java:662)
	at org.testcontainers.utility.ResourceReaper.start(ResourceReaper.java:142)
	... 34 more
```

Process finished with exit code -1


### Proposed Solution
<!-- Mandatory -->

### Possible Alternatives
<!-- Optional -->

### Additional Information
<!-- Optional -->

--- a/sormas-backend/src/test/resources/testcontainers/Dockerfile
+++ b/sormas-backend/src/test/resources/testcontainers/Dockerfile
@@ -1,7 +1,7 @@
 FROM postgres:10.18-alpine
 
 RUN apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/v3.13/main/ musl=1.2.2-r1 musl-dev=1.2.2-r1 && \
-    apk add --no-cache openssl=1.1.1n-r0 curl=7.79.1-r1 tzdata=2021a-r0 py-pip=20.3.4-r1 python3-dev=3.9.5-r2  \
+    apk add --no-cache openssl=1.1.1o-r0 curl=7.79.1-r1 tzdata=2021a-r0 py-pip=20.3.4-r1 python3-dev=3.9.5-r2  \
     postgresql-dev=13.6-r0 postgresql-contrib=13.6-r0 make=4.3-r0 gcc=10.3.1_git20210424-r2 py3-psutil=5.8.0-r1  && \
     pip install --no-cache-dir pgxnclient==1.3.2 && \
     pgxnclient install temporal_tables

--- a/sormas-base/pom.xml
+++ b/sormas-base/pom.xml
@@ -794,7 +794,7 @@
 			<dependency>
 				<groupId>org.testcontainers</groupId>
 				<artifactId>postgresql</artifactId>
-				<version>1.16.3</version>
+				<version>1.17.1</version>
 				<scope>test</scope>
 			</dependency>
 

